src/Search.js

```diff
@@ -1,30 +1,79 @@
 import React from "react";
 import "./App.css";
+import "./Search.css";
+import iFixitApi from './iFixitApi';
+import Autosuggest from 'react-autosuggest';
+import { FaSearch } from 'react-icons/fa';
+
+const getSuggestionValue = suggestion => suggestion.display_title;
+
+const renderSuggestion = suggestion => (
+	<span>{suggestion.display_title}</span>
+);
+
+const renderInputComponent = inputProps => (
+	  <div className="inputContainer">
+	    <FaSearch className="icon" />
+	    <input {...inputProps} />
+	  </div>
+);
 
 class SearchBar extends React.Component{
 	constructor(props){
 		super(props);
-		this.state = {searchString : ''};
+		this.state = {value : '', suggestions:[]};
 
-		this.handleChange = this.handleChange.bind(this);
+		this.onChange = this.onChange.bind(this);
```

Comment:
Since you used an arrow function for `this.onChange`, you don't need to bind `this` to it. This is actually a common pattern in React components used to avoid the extra bind call. I found this little blog post which goes into more details: https://medium.com/@joespinelli_6190/using-arrow-functions-to-avoid-binding-this-in-react-5d7402eec64

If you want you could refactor the other methods to use arrow functions so you don't have to bind `this` in the constructor.